json/src/main/java/com/basistech/rosette/apimodel/jackson/EntityMentionMixin.java

```diff
@@ -19,15 +19,17 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class EntityMixin extends BaseMixin {
+public class EntityMentionMixin extends BaseMixin {
     @JsonCreator
-    public EntityMixin(
+    public EntityMentionMixin(
             @JsonProperty("indocChainId") Integer indocChainId,
             @JsonProperty("type") String type,
             @JsonProperty("mention") String mention,
             @JsonProperty("normalized") String normalized,
-            @JsonProperty("count") int count,
-            @JsonProperty("entityId") String entityId
+            @JsonProperty("count") Integer count,
```

Comment:
@benson-basis I just realize that it we are no longer deduping the mentions, then this count makes no sense.